tests/tests_pytorch/utilities/test_data.py

```diff
@@ -311,14 +313,14 @@ def test_replace_dataloader_init_method(cls, args, kwargs, arg_names, dataset, c
         if isinstance(dataloader_value, torch.Tensor):
             assert dataloader_value is value
         else:
-            assert getattr(dataloader, key) == value
+            assert dataloader_value == value
```

Comment:
why does this use `==` vs `is` based on the type?

Reply:
Because checking tensor for equality using == requires `all()`. And since I am keeping the reference, because the data is random, it is easier to check for object equality. Btw, this was there from previous PR. Let me know, if you think I should change it to something else.